Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -29,30 +27,40 @@ namespace EdFi.Ods.Api.Helpers
     public static class AssemblyLoaderHelper
     {
         private static readonly ILog _logger = LogManager.GetLogger(typeof(AssemblyLoaderHelper));
-        private static readonly Dictionary<bool, bool> _assembliesAlreadyLoadedByIncludeFrameworkOption = new();
+        private static readonly Dictionary<Tuple<string, bool>, bool> _assembliesAlreadyLoadedByIncludeFrameworkOption = new();

```

Comment:
`Tuple` is a reference type. They've more recently introduced `ValueTuple` which is a value type, and language-specific syntax in C# for using them which provides for more code clarity than using the types explicitly. Consider declaring this as:
```csharp
private static readonly Dictionary<(string path, bool includeFramework), bool> _assembliesAlreadyLoadedByIncludeFrameworkOption = new();
```
Then the code editor will have the semantic information needed to provide more useful inline help, like so:
![image](https://github.com/user-attachments/assets/5b2be2ff-08c6-43ea-8294-e6378f8946c6)